src/App.js

```diff
@@ -1,41 +1,20 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Footer } from "./components/Footer/Footer";
+import Nav from "./components/Header/Nav";
+import Home from "./pages/Home/Home";
+import {Routes, Route} from 'react-router-dom';
+import { ProductListing } from "./pages/ProductListing/ProductListing";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+     <Nav/>
+    <Routes>
+      <Route path="/" element={<Home/>} ></Route>
+      <Route path="/product" element={<ProductListing/>}></Route>
```

Comment:
Suggestion : Make one route component and render it on APP comp, it looks clean to app comp when more routes added.

Reply:
thanks for pointing it out, will do that.